patterns/grok-patterns

```diff
@@ -32,6 +32,7 @@ HOSTPORT %{IPORHOST}:%{POSINT}
 # paths
 PATH (?:%{UNIXPATH}|%{WINPATH})
 UNIXPATH (?>/(?>[\w_%!$@:.,~-]+|\\.)*)+
+UNIXPATH (?>/(?>[\w_%!$@:.,~-=]+|\\.)*)+
```

Comment:
why don't you simply add the new character in the old path? this should be there. Duplicated paths would cause problems here.